### PR TITLE
feat(route): add Transcript Forest

### LINKED ISSUE #14462: [新订阅源]transcript forest
### 类型

社交媒体

### 网站地址

https://www.transcriptforest.com/

### 网站描述

带 transcript 的 Podcast

### 需要生成什么内容？

希望能订阅对应的 channel 信息。如订阅 all-the-hacks频道 https://www.transcriptforest.com/en/channel/all-the-hacks

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## Involved Issue / 该 PR 相关 Issue

Close #14462

## Example for the Proposed Route(s) / 路由地址示例

```routes
/transcriptforest/all-the-hacks
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: website/docs/routes/multimedia.mdx [hunk 1]
```diff
@@ -1135,6 +1135,65 @@ Refer to [https://developers.themoviedb.org/3/getting-started/languages](https:/
 
 <Route author="hoilc" example="/trakt/collection/tomyangsh/movies" path="/trakt/collection/:username/:type?" paramsDesc={['Username','Collection type, can be `movies`,`shows`,`episodes`,`all`, default to `all`']} radar="1" />
 
+## Transcript Forest {#transcript-forest}
+
+### Channel {#transcript-forest-channel}
+
+<Route author="nczitzk" example="/transcriptforest/all-the-hacks" path="/transcriptforest" paramsDesc={['Channel, see below, all by default']} radar="1" supportBT="1" supportPodcast="1">
```
Comment (TonyRL) [inline website/docs/routes/multimedia.mdx:1142]:
```suggestion
<Route author="nczitzk" example="/transcriptforest/all-the-hacks" path="/transcriptforest" paramsDesc={['Channel, see below, all by default']} radar="1" supportPodcast="1">
```

File: lib/v2/transcriptforest/templates/description.art [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <object data="{{ audio.src }}">
+          <embed src="{{ audio.src }}">
+        </object>
+      </audio>
+    {{ /if }}
+  {{ /each }}
+{{ /if }}
+
+{{ if description }}
+  <p>{{ description }}</p>
```
Comment (TonyRL) [inline lib/v2/transcriptforest/templates/description.art:17]:
```suggestion
  <p>{{@ description.replaceAll('\n', '<br>') }}</p>
```

File: lib/v2/transcriptforest/index.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        },
+    });
+
+    const items = response.pageProps.listEpisode.data.slice(0, limit).map((item) => ({
+        title: item.episode_name,
+        link: new URL(`${defaultLocale}/${item.channel_id}/${item.episode_id}`, rootUrl).href,
+        description: art(path.join(__dirname, 'templates/description.art'), {
+            audios: [
+                {
+                    src: item.source_media,
```
Comment (TonyRL) [inline lib/v2/transcriptforest/index.js:38]: This audio track contains ads.

To retrieve audio track w/o ads, you need to use `pageProps.currentEpisode.media` from `${rootUrl}/_next/data/${buildId}/${defaultLocale}/${item.channel_id}/${item.episode_id}.json` and the url to transcript is `pageProps.currentEpisode.ps4_url`

Comment (nczitzk) [reply lib/v2/transcriptforest/index.js:38]: Thanks for the review. 😊

**Pull request merged**